package.json

```diff
@@ -21,7 +21,7 @@
     "prebuild": "npm run clean",
     "build": "cross-env NODE_ENV=production babel src -d dist --copy-files",
     "commitlint": "commitlint --from=master",
-    "security": "npm audit",
+    "security": "npm audit --audit-level=moderate --parseable --production",
```

Comment:
Remove all flags

Reply:
Can you explain? That'll make this patch a full no-op and audit will be broken still.

Reply:
Acorn release new version with fixed, we don't need it anymore, just update lock file

Reply:
That wouldn't seem to help due to how things are pinned in Jest's JSDOM:

```
$ rm package-lock.json

$ npm i
added 10 packages from 84 contributors, removed 619 packages, updated 55 packages and audited 894519 packages in 18.582s
found 7 moderate severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details

$ npm audit; npm audit --parseable
=== npm audit security report ===

# Run  npm install --save-dev jest@25.1.0  to resolve 7 vulnerabilities
SEMVER WARNING: Recommended action is a potentially breaking change

[... snip ...]

found 7 moderate severity vulnerabilities in 894519 scanned packages
  7 vulnerabilities require semver-major dependency updates.


install	acorn	moderate	npm install --save-dev jest@25.1.0	Regular Expression Denial of Service	https://npmjs.com/advisories/1488	jest>jest-cli>@jest/core>@jest/reporters>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn	Y
install	acorn	moderate	npm install --save-dev jest@25.1.0	Regular Expression Denial of Service	https://npmjs.com/advisories/1488	jest>jest-cli>@jest/core>jest-runner>jest-jasmine2>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn	Y
install	acorn	moderate	npm install --save-dev jest@25.1.0	Regular Expression Denial of Service	https://npmjs.com/advisories/1488	jest>jest-cli>@jest/core>jest-runner>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn	Y
install	acorn	moderate	npm install --save-dev jest@25.1.0	Regular Expression Denial of Service	https://npmjs.com/advisories/1488	jest>jest-cli>@jest/core>jest-runtime>jest-config>jest-environment-jsdom>jsdom>acorn	Y
install	acorn	moderate	npm install --save-dev jest@25.1.0	Regular Expression Denial of Service	https://npmjs.com/advisories/1488	jest>jest-cli>@jest/core>jest-runner>jest-config>jest-environment-jsdom>jsdom>acorn	Y
install	acorn	moderate	npm install --save-dev jest@25.1.0	Regular Expression Denial of Service	https://npmjs.com/advisories/1488	jest>jest-cli>@jest/core>jest-config>jest-environment-jsdom>jsdom>acorn	Y
install	acorn	moderate	npm install --save-dev jest@25.1.0	Regular Expression Denial of Service	https://npmjs.com/advisories/1488	jest>jest-cli>jest-config>jest-environment-jsdom>jsdom>acorn	Y

$
```

Reply:
So @evilebottnawi, what should we do?

Reply:
Ignore it, we can't do something here